src/main/java/org/candlepin/insights/inventory/ConduitFacts.java

```diff
@@ -20,149 +20,58 @@
  */
 package org.candlepin.insights.inventory;
 
+import org.candlepin.insights.api.model.ConsumerInventory;
 import org.candlepin.insights.validator.ip.IpAddress;
 
 import org.hibernate.validator.constraints.Length;
 
 import java.util.List;
 
+import javax.validation.Valid;
 import javax.validation.constraints.Pattern;
 import javax.validation.constraints.Positive;
 
 /**
- * POJO that holds all facts scoped for collection by the conduit.
+ * POJO that validates all facts scoped for collection by the conduit.
  */
-public class ConduitFacts {
-
-    private String subscriptionManagerId;
-    private String biosUuid;
-    private String orgId;
-
-    private List<@IpAddress String> ipAddresses;
+public class ConduitFacts extends ConsumerInventory {
 
     @Length(min = 1, max = 255)
-
-    private String fqdn;
-
-    // See https://stackoverflow.com/a/4260512/6124862
-    // Also a soft validation.  A mixed delimiter MAC like a1:b2-c3:d4-e5:f6 will still validate
-    private List<@Pattern(regexp = "^([0-9A-Fa-f]{2}[:-]){5}([0-9A-Fa-f]{2})$") String> macAddresses;
-
-    @Positive
-    private Integer cpuSockets;
-
-    @Positive
-    private Integer cpuCores;
-
-    @Positive
-    private Integer memory;
-
-    private String architecture;
-    private Boolean isVirtual;
-    private String vmHost;
-    private List<String> rhProd;
-
-    public String getSubscriptionManagerId() {
-        return subscriptionManagerId;
-    }
-
-    public void setSubscriptionManagerId(String subscriptionManagerId) {
-        this.subscriptionManagerId = subscriptionManagerId;
-    }
-
-    public String getBiosUuid() {
-        return biosUuid;
-    }
-
-    public String getOrgId() {
-        return orgId;
-    }
-
-    public void setOrgId(String orgId) {
-        this.orgId = orgId;
-    }
-
-    public void setBiosUuid(String biosUuid) {
-        this.biosUuid = biosUuid;
-    }
-
-    public List<String> getIpAddresses() {
-        return ipAddresses;
-    }
-
-    public void setIpAddresses(List<String> ipAddresses) {
-        this.ipAddresses = ipAddresses;
-    }
-
+    @Override
     public String getFqdn() {
-        return fqdn;
+        return super.getFqdn();
     }
 
-    public void setFqdn(String fqdn) {
-        this.fqdn = fqdn;
+    // This is a soft validation.  Bogus IP addresses like "999.999.999.999" will still validate
```

Comment:
I don't know that this comment is accurate anymore!  @mstead used [InetAddresses isInetAddress](https://google.github.io/guava/releases/19.0/api/docs/com/google/common/net/InetAddresses.html#isInetAddress(java.lang.String)) method and it returns false for addresses like "999.3.3.3" (which is in the IpAddressValidatorTest.testInvalidIpV4Ip unit test).

Reply:
It certainly isn't. I sloppily pulled this back in from when I rebased my changes. Will address in a follow-up. Thanks for catching!